### ISSUE TITLE
Feature Request: Add tag friendly version for SPM (Swift Package Manager)

### ISSUE THREAD

**Issue by abrahamduran** [facebook/facebook-ios-sdk#2022]
### Checklist before submitting a feature request

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [Github issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [X] I've read the [Code of Conduct](https://github.com/facebook/facebook-ios-sdk/blob/main/CODE_OF_CONDUCT.md)

### Goals

When fetching the `facebook-ios-sdk` through SPM, I want to be able to see the latest version of the package, so I can do proper versioning.

SPM relies on a semantic version tag which has no prefixes (like `vX.X.X`). SPM will only recognize tags like `X.X.X`. Which is why the last one it recognizes is `4.36.0`.

### Expected results

I expect to see the latest version of the SDK when choosing `upToNextMajor` or `upToNextMinor`.

<img width="444" alt="image" src="https://user-images.githubusercontent.com/9121130/153097078-48f1b5fe-edce-4cad-b2fb-4c3acbb47b6e.png">

### Code samples & details

```swift
// INSERT YOUR CODE HERE
var example = "Example code"
```

**Comment by jawwad** [comment]
@abrahamduran -  I tried this in a new project and I am able to see the version tagged as v12.3.2 (and which does not have a 12.3.2 tag).

<img width="455" alt="image" src="https://user-images.githubusercontent.com/202240/153099268-a3ce11da-0b05-460b-95f8-5c683cdb9375.png">

Maybe this is some kind of caching issue with SPM?

Could you try removing then clearing the cache with the following and then re-adding SPM and see if the version number shows up?
```
rm -rf ~/Library/Caches/org.swift.swiftpm
rm -rf ~/Library/Developer/Xcode/DerivedData
```

**Comment by jawwad** [comment]
@abrahamduran - Actually my previous test was with Xcode 12, but I do see this behavior with Xcode 13.

<img width="512" alt="image" src="https://user-images.githubusercontent.com/202240/153099846-626f6136-3845-48c3-9997-c0e7fe2dc6a3.png">

I've tagged v12.3.1 as "12.3.1" and have pushed this tag. It hasn't shown up yet so maybe it takes some time to propagate. Let's see if 12.3.1 shows up in SPM.

**Comment by jawwad** [comment]
@abrahamduran - Can you check to see if the 12.3.1 tag is showing up for you?

I pushed the tag yesterday but it is still not showing up for me in Xcode 13
https://github.com/facebook/facebook-ios-sdk/releases/tag/12.3.1

Also, could you point me towards where you found that SPM won't recognize tags prefixed with v?

**Comment by abrahamduran** [comment]
Hey @jawwad of course, I got from the docs https://developer.apple.com/documentation/xcode/publishing_a_swift_package_with_xcode

> A package version is a three period-separated integer. An example is 1.0.0.

And also here: https://github.com/apple/swift-package-manager/blob/main/Documentation/Usage.md#publishing-a-package

Tho they don't explicitly say that "they don't allow prefixes", their only examples uses no prefixes.

Perhaps they changed the behavior in Xcode 13?

**Comment by jawwad** [comment]
Another observation. In Xcode 13.0.0 it shows 9.3.0 as the latest tag

<img width="1118" alt="image" src="https://user-images.githubusercontent.com/202240/153432972-3b0e8cef-8318-4c0e-85a0-66fa5424e5a5.png">

I found an article that shows there might be some issues with Xcode 13.2 in finding dependencies, but it doesn't mention this issue specifically.
https://appleinsider.com/articles/21/12/14/app-store-version-of-xcode-132-causing-problems-for-developers

**Comment by jawwad** [comment]
Based on my testing, I don't think this is an issue that we can fix on the SDK side since the "12.3.1" tag that I added without the v prefix is also not showing up. From the different behavior in Xcode 13.0 and Xcode 13.2 as well as the link in the previous comment this seems to be an issue with Xcode itself.
